kep/9-capacity-scheduling-for-ml-dl-workloads-based-on-scheduler-framework/README.md

```diff
@@ -70,24 +73,24 @@ updates.
 
 ## Motivation
 
-There is increasing demand to use Kubernetes to manage batch workloads (ML/DL). In those cases, one challenge is to improve cluster utilization while ensuring that each user has a reasonable amount of resources. The problem can be partially addressed by the Kubernetes ResourceQuota. The native Kubernetes ResourceQuota API can be used to specify the maximum overall resource allocation per namespace. The quota enforcement is done through an admission check. A quota resource consumer (e.g., a Pod) cannot be created if the aggregated resource allocation exceeds the quota limit. The Kubernetes quota design has the following limitations:
+There is increasing demand to use Kubernetes to manage batch workloads (ML/DL). In those cases, one challenge is to improve cluster utilization while ensuring that each user has a reasonable amount of resources. The problem can be partially addressed by the Kubernetes [ResourceQuota](https://kubernetes.io/docs/concepts/policy/resource-quotas/). The native Kubernetes ResourceQuota API can be used to specify the maximum overall resource allocation per namespace. The quota enforcement is done through an admission check. A quota resource consumer (e.g., a Pod) cannot be created if the aggregated resource allocation exceeds the quota limit. The Kubernetes quota design has the limitation: the quota resource usage is aggregated based on the resource configurations (e.g., Pod cpu/mem requests specified in the Pod spec). Although this mechanism can guarantee that the actual resource consumption will never exceed the ResourceQuota limit, it might lead to low resource utilization. For instance, actual resource 
+consumption may be much smaller than the limit.
 
-1. The quota resource usage is aggregated based on the resource configurations (e.g., Pod cpu/mem requests specified in the Pod spec). Although this mechanism can guarantee that the actual resource consumption will never exceed the ResourceQuota limit, it might lead to low resource utilization had the actual resource consumption been much smaller than the limit. i.e., lead to internal resource fragmentation.
+Due to above limitation, the batch workloads (ML/DL) can't run in a Kubernetes cluster as efficiently as they do in other container orchestration platforms such as Yarn. In order to overcome above limitations, we introduce an "ElasticQuota" concept used in [Yarn capacity scheduler](https://www.netjstech.com/2018/04/capacity-scheduler-in-yarn-hadoop.html) in Kubernetes. Basically, the "ElasticQuota" has the notions of "max" and "min", where the "min” is the minimum resources that are needed to ensure the basic functionality/performance of the consumers and the "max" specifies the upper bound of the resource consumption of the consumers. By introducing "min" and "max", capacity scheduling can be optimized as follows:
```

Comment:
"min” -> "min"

Reply:
I think the term "capacity scheduling" and "Resource Quota" should be officially defined, or you may want to declaim that these two terms are used interchangeably.

Reply:
Or, you may want to say something like:

```
We proposed a new "ResourceQuota" mechanics to optimize current capacity scheduling as follows:
```

Reply:
In order to overcome above limitations, we introduce an "ElasticQuota" concept used in
->
In order to overcome above limitations, an "ElasticQuota" concept used in
Yarn can be leveraged into Kubernetes.

Reply:
Move the description of "min" and "max" to the first bullet. And here, maybe say something like:

```
Basically, the "ElasticQuota" aims to provide basic quota management
while also providing flexibility on a certain extend of overselling.
```

Reply:
Done